docs/sources/auth/overview.md

```diff
@@ -125,6 +125,10 @@ Defaults to `false`.
 oauth_auto_login = true
 ```
 
+If you want to skip automatic OAuth login and sign in with username and password, add `disableAutoLogin` parameter to the login URL. Signing in with OAuth will be disabled in this case.
+
+Example: `grafana.example.com/login?disableAutoLogin` or `grafana.example.com/login?disableAutoLogin=true`.
```

Comment:
```suggestion
Example: `grafana.example.com/login?disableAutoLogin` or `grafana.example.com/login?disableAutoLogin=true`
```

Reply:
Great examples! We need more examples throughout the doc. I think they really help clarify items like this.